Source/ZKApp.cpp

```diff
@@ -389,7 +390,7 @@ ZooKeeperApp::MakeTempFolder(void)
 
 	status_t status;
 	BPath p;
-	if (find_directory(B_COMMON_TEMP_DIRECTORY, &p) != B_OK)	return B_ERROR;
+	if (find_directory(B_SYSTEM_TEMP_DIRECTORY, &p) != B_OK)	return B_ERROR;
```

Comment:
While you're updating constants, can you correct the path to Terminal in this file, line 368? It's "`/boot/system/apps/Terminal -t`"